### DISCUSSION TITLE
upload组件，上传失败后，默认提示的失败信息怎样关闭？

### DISCUSSION THREAD

**Discussion by FlankMiao** [layui/layui#2345] [问答] [answered]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

2.9.17

### 问题描述

upload上传失败时，会有一个默认的信息提示，自定义error方法后，默认的还是存在，怎样关闭这个默认的失败信息提示呢

### 业务代码

```auto
upload上传失败时，会有一个默认的信息提示，自定义error方法后，默认的还是存在，怎样关闭这个默认的失败信息提示呢？text属性中error属性，只能改变提示的信息
```

### 浏览器

 129.0.6668.90

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment, marked as answer]
无法关闭。但可自定义失败提示文本。
https://layui.dev/docs/2/upload/#options.text

**Comment by FlankMiao** [comment]
好的，谢谢，希望有自定义error方法后就不再显示默认的失败提示信息。
